### PR TITLE
Adding alloys

### PR DESCRIPTION
**Pull request by siaynoq**

### CODE REVIEW

**Comment by siaynoq** [comment]
Hey @TheBusyBiscuit  - any comment for this PR? :)

**Comment by TheBusyBiscuit** [comment]
ehhhh, ugh, this one went completely under my radar somehow. Thanks for reminding me, I'll have a look later or tomorrow when I can.

File: pages/Redstone-Alloy-Ingot.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+## Obtaining
+
+Combining the following items in a [Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Smeltery) or an [Electric Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Smeltery) yields one Hardened Redstone Alloy Ingot:
```
Comment (TheBusyBiscuit) [inline pages/Redstone-Alloy-Ingot.md:4]: *Hardened* Redstone Alloy?

Comment (siaynoq) [reply pages/Redstone-Alloy-Ingot.md]: Oops :) Fixed!

**Pull request merged**